molecule/rbac-mds-kerberos-debian/molecule.yml

```diff
@@ -81,8 +81,8 @@ platforms:
     groups:
       - ${CONTROLLER_HOSTGROUP:-zookeeper}2
       - cluster2
-    image: geerlingguy/docker-debian9-ansible
-    dockerfile: ../Dockerfile-debian.j2
+    image: geerlingguy/docker-debian10-ansible
+    dockerfile: ../Dockerfile-debian10-archive.j2
```

Comment:
why are we using Dockerfile-debian10-archive here? This is not an archive scenario

Reply:
Actually `Dockerfile-debian10.j2` and `Dockerfile-debian10-archive.j2` only differ in one command. `Dockerfile-debian10-archive.j2` has extra command to `apt-key add & add-apt-repository`. So I didnt think it would matter whichever we would use. But to remain consistent with naming will change the Dockerfile in this scenario.